### PR TITLE
Episode 19 Part1 + Part2

### PR DESCRIPTION
**Pull request by sockeqwe**
- Feed is [valid](http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode19_part1/feed.rss)

TODO's:

- [x] Upload mp3 file `The.Context.episode.19.part1.mp3`
- [x] Upload mp3 file `The.Context.episode.19.part2.mp3` (Part 2)
- [ ]  Add git tag
- [ ] Upload mp3 files in release section

### CODE REVIEW

File: show_notes/Episode_19_Part1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/95)
+
+In the first part of episode #19 Artur and Hannes chatted with Benoît Quenaudon from Square about Model-View-Intent. 
+We discussed the difference between MVP, MVVM and MVI, how to implement MVI, reusable components and how to test MVI based applications.
+
+### Guest:
+- Benoît Quenaudon [@artem_zin](https://twitter.com/oldergod), [personal blog](https://benoitquenaudon.com), [GitHub](https://github.com/oldergod)
```
Comment (artem-zinnatullin) [inline show_notes/Episode_19_Part1.md]: wrong twitter text in url

File: feed.rss [hunk 2]
```diff
@@ -1373,6 +1373,55 @@
...
+				<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/95">Discussion after the episode</a></li>
+				</ul>
+
+				<p>In the first part of episode #19 Artur and Hannes chatted with Benoît Quenaudon from Square about Model-View-Intent. 
+				We discussed the difference between MVP, MVVM and MVI, how to implement MVI, reusable components and how to test MVI based applications.</p>
+
+				<h3>Guest:</h3>
+
+				<ul>
+				<li>Benoît Quenaudon <a href="https://twitter.com/oldergod">@artem_zin</a>, <a href="https://benoitquenaudon.com">personal blog</a>, <a href="https://github.com/oldergod">GitHub</a></li>
```
Comment (artem-zinnatullin) [inline feed.rss]: wrong twitter text

File: show_notes/Episode_19_Part2.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  
+
+#### Links
+
+- [Cycle.js](https://cycle.js.org)
+- [Android Blueprint: ToDo App using MVI by Benoît](https://github.com/oldergod/android-architecture)
+- [droidcon NYC 2017 - Model-View-Intent for Android](https://www.youtube.com/watch?v=PXBXcHQeDLE)
+- [In-App Navigation with Coordinators](http://hannesdorfmann.com/android/mosby3-mvi-8)
+- [droidcon SF 2017 - The Reactive Workflow Pattern Update](https://www.youtube.com/watch?v=mvBVkU2mCF4)
+- [Benoît's Tweet](https://twitter.com/oldergod/status/999638960384233474): "We see more and more "reactive" architectures on Android but I think you're doing it wrong if you have a `subscribe` in your presenter. Ideally, the presenter should be an ObservableTransformer."
```
Comment (ming13) [inline show_notes/Episode_19_Part2.md]: I think we can add this link — https://staltz.com/unidirectional-user-interface-architectures.html

Comment (sockeqwe) [reply show_notes/Episode_19_Part2.md]: Added 👍

**Pull request merged**